tests/core/pyspec/eth2spec/test/phase0/unittests/fork_choice/test_on_block.py

```diff
@@ -192,21 +95,33 @@ def test_on_block_update_justified_checkpoint_within_safe_slots(spec, state):
 
     next_epoch(spec, state)
     spec.on_tick(store, store.time + state.slot * spec.config.SECONDS_PER_SLOT)
-    state, store, last_signed_block = apply_next_epoch_with_attestations(spec, state, store)
+    state, store, last_signed_block = yield from apply_next_epoch_with_attestations(spec, state, store)
     next_epoch(spec, state)
     spec.on_tick(store, store.time + state.slot * spec.config.SECONDS_PER_SLOT)
-    last_block_root = hash_tree_root(last_signed_block.message)
+    last_block = last_signed_block.message
```

Comment:
Why the intermediate variable `last_block`? it doesn't seem to be used

Reply:
You are right, `last_block` was only used for debugging previously. Now it can be removed.